### PR TITLE
OME Files 0.2.1

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
Update download page and version numbers

### CODE REVIEW

File: files_cpp_downloads.html [hunk 1]
```diff
@@ -252,11 +231,24 @@
...
+                   platform release and therefore should always work
+                   with the shared libraries provided by the platform.
+                   These releases are provided only for platforms with
+                   guaranteed ABI stability during their supported
+                   lifetime, and which also provide all of the needed
+                   build dependencies.
+                </p>
+                <p>The system libraries required by the OME Files
+                   libraries must be installed.  These are listed in
+                   the <a href="@DOC_SUPERBUILD_URL@prerequisites.html">requirements</a>.
```
Comment (mtbc) [inline files_cpp_downloads.html:246]: maybe only those from the "deploy" column on `prerequisites/ome-files`?

**Comment by hflynn** [comment]
Will review tomorrow once there's a build in place to look at the rendered page.

**Comment by hflynn** [comment]
👍 to moving the build explanatory text. Discussed some further changes to the very top and the doc links with Roger, will look again once they are implemented.

**Comment by hflynn** [comment]
👍 to page changes from me

**Pull request merged**